example/e2e/maestro/simple-stack.yml

```diff
@@ -39,4 +39,4 @@ name: Simple Stack
     text: 'Pop by 2'
     retryTapIfNoChange: false
 - assertVisible:
-    text: 'Contacts'
+    text: 'Article by Babel fish'
```

Comment:
Previously step 11 

```
- tapOn:
    text: 'Push article'
    retryTapIfNoChange: false
```

Would change the `article` screen from index 0 to index 2.

After this change, `article` would remain at index 0 and a new route will be pushed. It will have an index of 3 and be the current screen